### ISSUE TITLE
Proposal: automate claiming

### ISSUE THREAD

**Issue by Lialee88** [graphprotocol/mission-control-curator#290]
When someone adds project on Everest - we can automatically send message to project owners (twitter, github, or specify other required project owner credentials).

**Comment by yanivtal** [comment]
Yes we need to add messaging to make claiming faster and more reliable.
